lang/typescript/src/utils/address-fields.ts

```diff
@@ -71,3 +72,33 @@ export function splitAddressField(
 
   return parsedAddressObject;
 }
+
+/**
+ * Utility function that parses an address string based on a provided field
+ * definition and set of values for those fields, and a regex pattern as a fallback
+ *
+ * @param regex Regex pattern for parsing out sub-fields
```

Comment:
Missing the first param for fieldDefinition here `@param fieldDefinition Array of definitions of address sub-fields`

Reply:
Thanks for catching that!